pallets/automation-time/src/lib.rs

```diff
@@ -304,6 +304,15 @@ pub mod pallet {
 			who: AccountOf<T>,
 			task_id: TaskId<T>,
 		},
+		TaskRescheduled {
```

Comment:
can you add a description?  `rescheduled` is kind of a confusing way to put it.  not sure I have a much better idea.  maybe `renewed`?   unless I'm misunderstanding what the event is for.

Reply:
I added a doc comment to help clear it up. Let me know what you think. I'm open to renaming it, but  also not sure on a good name. I'm not sure on `renew` will think on it.